api-specificatie/ztc/1.0.x/openapi.yaml

```diff
@@ -1,129 +1,42 @@
-
-
-
-    Deze API publiceert notificaties op het kanaal `besluittypen`.
-
-
-    **Main resource**
-
-
-    `besluittype`
-
-
-
-
-    **Kenmerken**
-
-
-    * `catalogus`: URL-referentie naar de CATALOGUS waartoe dit BESLUITTYPE behoort.
-
-
-    **Resources en acties**
-
-    - `besluittype`: create, update, destroy
-
-
-
-    Deze API publiceert notificaties op het kanaal `informatieobjecttypen`.
-
-
-    **Main resource**
-
-
-    `informatieobjecttype`
-
-
-
-
-    **Kenmerken**
-
-
-    * `catalogus`: URL-referentie naar de CATALOGUS waartoe dit INFORMATIEOBJECTTYPE
-    behoort.
-
-
-    **Resources en acties**
-
-    - `informatieobjecttype`: create, update, destroy
-
-
-
-    **Handige links**
-
-
-    * [Documentatie](https://zaakgerichtwerken.vng.cloud/standaard)
-
-    * [Zaakgericht werken](https://zaakgerichtwerken.vng.cloud)
-
-    '
+  description: "Een API om een zaaktypecatalogus (ZTC) te benaderen.\n\nDe zaaktypecatalogus\
+    \ helpt gemeenten om het proces vanuit de 'vraag van een\nklant' (productaanvraag,\
+    \ melding, aangifte, informatieverzoek e.d.) tot en met\nhet leveren van een passend\
+    \ antwoord daarop in te richten, inclusief de\nbijbehorende informatievoorziening.\n\
+    \nEen CATALOGUS bestaat uit ZAAKTYPEn, INFORMATIEOBJECTTYPEn en BESLUITTYPEn en\n\
+    wordt typisch gebruikt om een ZAAK (in de Zaken API), INFORMATIEOBJECT (in de\n\
+    Documenten API) en BESLUIT (in de Besluiten API) te voorzien van type,\nstandaardwaarden\
+    \ en processtructuur.\n\n**Afhankelijkheden**\n\nDeze API is afhankelijk van:\n\
+    \n* Gemeentelijke Selectielijst API\n* Autorisaties API *(optioneel)*\n\n\n**Autorisatie**\n\
+    \nDeze API vereist autorisatie. Je kan de\n[token-tool](https://zaken-auth.vng.cloud/)\
+    \ gebruiken om een token te\ngenereren.\n\n_Zelf een token genereren_\n\nDe tokens\
+    \ die gebruikt worden voor autorisatie zijn [jwt.io][JWT's] (JSON web\ntoken).\
+    \ In de API calls moeten deze gebruikt worden in de `Authorization`\nheader:\n\
+    \n```\nAuthorization: Bearer <token>\n```\n\nOm een JWT te genereren heb je een\
+    \ `client ID` en een `secret` nodig. Het JWT\nmoet gebouwd worden volgens het\
+    \ `HS256` algoritme. De vereiste payload is:\n\n```json\n{\n    \"iss\": \"<client\
+    \ ID>\",\n    \"iat\": 1572863906,\n    \"client_id\": \"<client ID>\",\n    \"\
+    user_id\": \"<user identifier>\",\n    \"user_representation\": \"<user representation>\"\
+    \n}\n```\n\nAls `issuer` gebruik je dus je eigen client ID. De `iat` timestamp\
+    \ is een\nUNIX-timestamp die aangeeft op welk moment het token gegenereerd is.\n\
+    \n`user_id` en `user_representation` zijn nodig voor de audit trails. Het zijn\n\
+    vrije velden met als enige beperking dat de lengte maximaal de lengte van\nde\
+    \ overeenkomstige velden in de audit trail resources is (zie rest API spec).\n\
+    \n\n### Notificaties\n\nDeze API publiceert notificaties op het kanaal `zaaktypen`.\n\
```

Comment:
Ik stel voor de 'Notificaties' sectie te vervangen door de volgende markdown de deze sectie iets beter ordent:

    \n\n### Notificaties\n\n\
    **zaaktypen**\n\n* Deze API publiceert notificaties op het kanaal `zaaktypen`.\n\
    * **Main resource**: `zaaktype`\n\
    * **Kenmerken**: `catalogus`: URL-referentie naar de CATALOGUS waartoe dit ZAAKTYPE behoort.\n\
    * **Resources en acties**: `zaaktype`: create, update, destroy\n\n\n\
    **besluittypen**\n\n* Deze API publiceert notificaties op het kanaal `besluittypen`.\n\
    * **Main resource**: `besluittype`\n\
    * **Kenmerken**: `catalogus`: URL-referentie naar de CATALOGUS waartoe dit BESLUITTYPE behoort.\n\
    * **Resources en acties**: `besluittype`: create, update, destroy\n\n\n\ 
    **informatieobjecttypen**\n\n* Deze API publiceert notificaties op het kanaal `informatieobjecttypen`.\n\
    * **Main resource**: `informatieobjecttype`\n\
    * **Kenmerken**: `catalogus`: URL-referentie naar de CATALOGUS waartoe dit INFORMATIEOBJECTTYPE behoort.\n\
    * **Resources en acties**: `informatieobjecttype`: create, update, destroy\n\
	
of 	

    \n\n### Notificaties\n\n\
    **zaaktypen**\n\n* Deze API publiceert notificaties op het kanaal `zaaktypen`.\n\
    * **Main resource**: `zaaktype`\n\
    * **Kenmerken**:\n\
    \n    `catalogus`: URL-referentie naar de CATALOGUS waartoe dit ZAAKTYPE behoort.\n\
    * **Resources en acties**:\n\
    \n    `zaaktype`: create, update, destroy\n\n\n\
    **besluittypen**\n\n* Deze API publiceert notificaties op het kanaal `besluittypen`.\n\
    * **Main resource**: `besluittype`\n\
    * **Kenmerken**:\n\
    \n    `catalogus`: URL-referentie naar de CATALOGUS waartoe dit BESLUITTYPE behoort.\n\
    * **Resources en acties**:\n\
    \n    `besluittype`: create, update, destroy\n\n\n\ 
    **informatieobjecttypen**\n\n* Deze API publiceert notificaties op het kanaal `informatieobjecttypen`.\n\
    * **Main resource**: `informatieobjecttype`\n\
    * **Kenmerken**:\n\
    \n    `catalogus`: URL-referentie naar de CATALOGUS waartoe dit INFORMATIEOBJECTTYPE behoort.\n\
    * **Resources en acties**:\n\
    \n    `informatieobjecttype`: create, update, destroy\n\

De laatste is beter als onder de kopjes 'Kenmerken' of 'Resources en acties' meerdere opties moeten worden vermeld wat nu niet het geval is.

Reply:
eh ik zie niet goed wat het verschil is, kan je dat in tekst verduidelijken?

Reply:
@melsk-r en sowieso beter om hier een aparte US voor te maken, heeft namelijk niet echt wat te maken met deze PR over deelzaaktypen

Reply:
Wat bedoeld wordt is een verschil in layout van de tekst, zie plaatje:
![image](https://user-images.githubusercontent.com/34736717/68394617-79d11f80-016e-11ea-9efe-2e8943ba9752.png)

Reply:
Het eerste is rommelig en minder overzichtelijk terwijl het tweede veel strakker en duidelijker is.

Inhoudelijk maakt het dus niets uit.

Reply:
Ach, op die fiets. Mee akkoord - de markdown templates hiervoor staan op:

* https://github.com/VNG-Realisatie/vng-api-common/blob/master/vng_api_common/notifications/kanalen.py#L62
* https://github.com/VNG-Realisatie/vng-api-common/blob/master/vng_api_common/notifications/utils.py#L9

PR zou welkom zijn! 😉